project/AppDependencies.scala

```diff
@@ -3,39 +3,38 @@ import sbt._
 
 object AppDependencies {
 
-  val mongoVersion = "0.68.0"
+  private val hmrcMongoPlayVersion = "0.71.0"
+  val scope                        = "test"
```

Comment:
move it above test seq or on top of the variables not in between versions it helps with readability